src/Admin/SnapshotAdmin.php

```diff
@@ -68,7 +68,7 @@ public function getBatchActions()
         $actions = parent::getBatchActions();
 
         $actions['toggle_enabled'] = [
-            'label' => $this->trans('toggle_enabled'),
+            'label' => $this->getTranslator()->trans('toggle_enabled', [], 'SonataPageBundle'),
```

Comment:
wouldn't this be automatic translated by the translator anyway?

Reply:
Indeed